src/Sulu/Bundle/CoreBundle/Cache/StructureWarmer.php

```diff
@@ -30,15 +30,25 @@ public function __construct(StructureManagerInterface $structureManager)
         $this->structureManager = $structureManager;
     }
 
+    /**
+     * Warms up the cache.
+     *
+     * @return string[] A list of classes or files to preload on PHP 7.4+
+     */
```

Comment:
I think it is better to use @inheritDoc here. The definition in Sf 4.4 differs from 5.5